tests/routes/emailShare.test.js

```diff
@@ -0,0 +1,95 @@
+import chai from 'chai';
+import sinon from 'sinon';
+import server from '../../index';
+import db from '../../models';
+import { AUTHORIZATION_HEADER } from '../../helpers/constants';
+import Authorization from '../../middlewares/Authorization';
+import { realUser1 } from '../testHelpers/testLoginData';
+import { defaultArticle } from '../testHelpers/articleUtil';
+import { deleteTable } from '../testHelpers';
+
+const { expect } = chai;
+const { User, Article } = db;
+
+describe('POST /share', () => {
+  let user, jwt, publicArticleSlug, privateArticleSlug, unpublishedArticleSlug;
+
+  before(async () => {
```

Comment:
It is sometimes advisable to add some sort of message to your test hook, this is to make it easy to debug once there is an error.

Reply:
Thank you. Mocha handles this out of the box by specifying that the error occurred in the `before` hook.